sellotape/sellotape_dj/main_app/models.py

```diff
@@ -1,3 +1,7 @@
-from django.db import models
-
-# Create your models here.
+from django.db import models
+
+
+class user_followers(models.Model):
+    user_followers_id 	=     user_id = models.AutoField(primary_key=True)

```

Comment:
You don't need to create the the primary key id field manually. From the [tutorial](https://docs.djangoproject.com/en/3.1/intro/tutorial02/):

> Primary keys (IDs) are added automatically. (You can override this, too.)